src/pyOpenMS/pxds/FeatureFinderAlgorithmMetaboIdent.pxd

```diff
@@ -0,0 +1,59 @@
+from Peak1D cimport *
+from Feature cimport *
+from FeatureMap cimport *
+from MSExperiment cimport *
+from Peak1D cimport *
+from ChromatogramPeak cimport *
+from FeatureFinder cimport *
+from DefaultParamHandler cimport *
+from TargetedExperiment cimport *
+from TransformationDescription cimport *
+
+cdef extern from "<OpenMS/TRANSFORMATIONS/FEATUREFINDER/FeatureFinderAlgorithmMetaboIdent.h>" namespace "OpenMS":
+
+    cdef cppclass FeatureFinderAlgorithmMetaboIdent(DefaultParamHandler):
+
+        # wrap-inherits:
+        #    DefaultParamHandler
+        # wrap-doc:
+        #   Perform targeted feature extraction of compounds provided as table and stores them in features.
+        #   -----
+        #   The algorithms detects quantitative features in MS1 data for a list of targets, typically small molecule/metabolite identifications.
+        #   Internally, it uses algorithms for targeted data analysis from the OpenSWATH pipeline.
+        #   In the simplest case, only CompoundName, SumFormula, Charge and RetentionTime need to be given, all other values may be zero.
+        #   Every combination of compound (mass), RT and charge defines one target for feature detection.
+        #   Output:
+        #   The main output is a feature map of detected features, with annotations in meta data entries.
+        #   Additional outputs are the extracted chromatograms/peak groups, the assay in TraML compatible format, and transformations 
+        #   that contain the error between provided and observed peaks.
+
```

Comment:
would be great to also have a short usage example here similar to:
https://github.com/OpenMS/OpenMS/blob/develop/src/pyOpenMS/pxds/MSExperiment.pxd#L37-L44